runtime/near-vm-runner/src/imports.rs

```diff
@@ -62,19 +62,15 @@ macro_rules! call_with_name {
 
 macro_rules! imports {
     (
-      $($(#[$stable_feature:ident])? $(#[$feature_name:literal, $feature:ident])* $(##[$feature_name2:literal])?
+      $($(#[$config_field:ident])? $(##[$feature_name:literal])?
         $( @in $mod:ident : )?
         $( @as $name:ident : )?
         $func:ident < [ $( $arg_name:ident : $arg_type:ident ),* ] -> [ $( $returns:ident ),* ] >,)*
     ) => {
         macro_rules! for_each_available_import {
-            ($protocol_version:expr, $M:ident) => {$(
-                $(#[cfg(feature = $feature_name2)])?
-                $(#[cfg(feature = $feature_name)])*
-                if true
-                    $(&& near_primitives_core::checked_feature!($feature_name, $feature, $protocol_version))*
-                    $(&& near_primitives_core::checked_feature!("stable", $stable_feature, $protocol_version))?
-                {
+            ($config:expr, $M:ident) => {$(
+                $(#[cfg(feature = $feature_name)])?
+                if true $(&& ($config).$config_field)? {
```

Comment:
Just to make sure I understand this right:

The old code uses a cfg condition to check for two cargo features, (1) for a nightly protocol feature and (2) for other features such as sandbox. Then it uses `checked_feature!` to also respect the protocol version.

The new code uses config flags and hence no longer needs the protocol feature check and is unaware of protocol versions. What remains is a `cfg` for sandbox and a simple check if `config.flag` is enabled.

If I got this right, it does make sense to me. But I want to make sure I'm not missing some subtlety.